digitaltwins/samples/device/javascript/sample_device.js

```diff
@@ -23,19 +31,19 @@ const propertyUpdateHandler = async (component, propertyName, reportedValue, des
 
 const commandHandler = (request, response) => {
   console.log('received command: ' + request.commandName + ' for component: ' + request.componentName);
-  response.acknowledge(200, 'helpful response text')
-    .then(() => console.log('acknowledgement succeeded.'));
+  await response.acknowledge(200, 'helpful response text');
+  console.log('acknowledgement succeeded.');
 };
 
 
 const environmentalSensor = new EnvironmentalSensor('environmentalSensor', propertyUpdateHandler, commandHandler);
 const deviceInformation = new DeviceInformation('deviceInformation');
 
-const capabilityModel = 'dtmi:azureiot:samplemodel;1';
+const rootInterfaceId = 'dtmi:contoso_device_corp:samplemodel;1';
 
 async function main() {
   // mqtt is implied in this static method
-  digitalTwinClient = DigitalTwinClient.fromConnectionString(capabilityModel, process.env.DEVICE_CONNECTION_STRING);
+  digitalTwinClient = DigitalTwinClient.fromConnectionString(rootInterfaceId, process.env.DEVICE_CONNECTION_STRING);
```

Comment:
We should avoid the term `Root interface` and use `ModelId` instead.

Reply:
@rido-min so are we using Root Interface Id or Device Capability Model?

Reply:
Device Capability Model -> Model (implemented by different interfaces)
DCM Id -> ModelId

There is no such concept as Root Interface in DTDL v2.

/cc @ericmitt @jspaith @jebrando

Reply:
this is the PR I am referencing: https://github.com/Azure/azure-iot-sdk-c/pull/1508

Reply:
@rido-min can you send @anthonyvercolano and I and email about this new spec, we were unaware of this change...